<?xml version="1.0"?>
<launch>
  <node name="identify_case" pkg="medicine_calendar" type="identify_case" output="screen">
    <param name="threshold_calendar" value="0.007"/> <!--値が小さいほど平面の条件が小さくなる-->
    <param name="calendar_number_neighbors" value="200"/>
    <param name="cluster_tolerance" value="0.01"/>
    <param name="max_cluster_size" value="100000"/>
    <param name="min_cluster_size" value="50000"/>
    <param name="threshold_case" value="0.012"/>
    <param name="case_number_neighbors" value="50"/>
    <param name="case_cluster_tolerance" value="0.01"/>
    <param name="max_case_cluster_size" value="1100"/>
    <param name="min_case_cluster_size" value="200"/>
    <param name="st_x" value="0.075"/>
    <param name="st_y" value="0.06"/>
    <param name="st_z" value="0.035"/>
    <param name="set_week" value="3"/>
    <param name="set_time" value="12"/>
  </node>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find medicine_calendar)/rviz/rsh_sample.rviz" required="true" output="screen" />
</launch>
